src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
+'use strict';
```

Comment:
The 'use strict' directive is repeated here and should be removed as it is already present at the beginning of the file.